src/main/java/io/dropwizard/revolver/persistence/AeroSpikePersistenceProvider.java

```diff
@@ -276,15 +277,19 @@ private RevolverCallbackRequest recordToRequest(Record record) {
         } catch (IOException e) {
             log.warn("Error decoding response", e);
         }
+        Map<String, List<String>> headersKeyIgnoreCaseMap = new StringKeyIgnoreCaseMultivaluedMap<>();
```

Comment:
It's failing while extracting the headers from map because of case insenstivity
```callbackRequest.getHeaders()
                    .getOrDefault(RevolversHttpHeaders.CALLBACK_METHOD_HEADER, Collections.singletonList("POST")).get(0);```